src/hotspot/share/gc/shenandoah/shenandoahScanRemembered.cpp

```diff
@@ -82,19 +82,25 @@ void ShenandoahDirectCardMarkRememberedSet::mark_range_as_clean(size_t card_inde
   }
 }
 
-bool ShenandoahDirectCardMarkRememberedSet::is_card_dirty(HeapWord *p) const {
+bool ShenandoahDirectCardMarkRememberedSet::is_card_dirty(HeapWord* p) const {
   size_t index = card_index_for_addr(p);
   CardValue* bp = &(_card_table->read_byte_map())[index];
   return (bp[0] == CardTable::dirty_card_val());
 }
 
-void ShenandoahDirectCardMarkRememberedSet::mark_card_as_dirty(HeapWord *p) {
+bool ShenandoahDirectCardMarkRememberedSet::is_write_card_dirty(HeapWord* p) const {
```

Comment:
Interesting. Is this new? Is it used somewhere or introduced for the purpose of a more consistent/uniform API?

(Not requiring any change; just asking. I am all for uniform/consistent APIs.)

Reply:
Never mind. Found the (new) place where it's now used.